server/monitor/src/main/java/org/apache/accumulo/monitor/next/InformationFetcher.java

```diff
@@ -270,7 +275,13 @@ public void run() {
       // Only refresh every 5s (old monitor logic).
       while (!newConnectionEvent.get() && connectionCount.get() == 0
           && NanoTime.millisElapsed(refreshTime, NanoTime.now()) > 5000) {
-        Thread.onSpinWait();
+        try {
+          Thread.sleep(100);
+        } catch (InterruptedException e) {
+          Thread.currentThread().interrupt();
+          throw new IllegalStateException(
+              "Thread " + Thread.currentThread().getName() + " interrupted", e);
+        }
```

Comment:
Will cause Monitor to terminate when the IllegalStateException occurs (this is thrown in the run method of a critical thread). Maybe there's some other way we want to handle this or maybe this is good.